client/components/button-group/index.jsx

```diff
@@ -9,7 +9,11 @@ const ButtonGroup = ( { busy = false, children, className = '', primary = false
 		'is-primary': primary,
 	} );
 
-	return <span className={ buttonGroupClasses }>{ children }</span>;
+	return (
+		<span className={ buttonGroupClasses } role="button">
```

Comment:
Sorry, but the button group doesn't make much sense to have a "button" role. Its children _could_ have that role, but it's not a requirement. The parent IMHO should have a `role="group"`. What's the case against that?